### ISSUE TITLE
GET /file_formats: Expose supported data types and media types

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-api#476]
For an easier way of understanding the exposed file formats, it would be helpful to give some more information on what they offer.

Right now we just expose the "gis_data_type" (raster, vector, table, ...), but it would be great to get some more information so that clients can easily hint at potential improvements, e.g. scaling.

Interesting could be:
- Media type (e.g. image/png)
- Data type (e.g. uint8 for PNG)
- Max number of bands (e.g. 1/3/4)

**Comment by soxofaan** [comment]
As discussed at https://github.com/Open-EO/openeo-processes/issues/498, explicitly linking file formats to media types seems a very valuable step here indeed, e.g. to unlock interoperability with other tools

**Comment by m-mohr** [comment]
One potential issue:

For example a GTIFF file format could have multiple media types, e.g. `image/tiff; application=geotiff` and `image/tiff; application=geotiff; cloud-optimized=true` depending on which (backend-specific) option has been set.

**Comment by soxofaan** [comment]
yes, but we could allow listing multiple possible media types per file format, right? Or do you see a problem there?

**Comment by m-mohr** [comment]
Depends on the usecases, I guess. We should probably document which usecases we try to solve here.

**Comment by m-mohr** [comment]
Moving to 1.4.0 as I don't see an urgent need for it and also it seems that it needs more discussions.
If discussions spin up in the next weeks and we have a conclusion or PR ready before the 1.3.0 release, happy to move it back to 1.3.0.
